palette/src/encoding/srgb.rs

```diff
@@ -42,9 +42,9 @@ impl LumaStandard for Srgb {
 impl TransferFn for Srgb {
     fn into_linear<T: Float + FromF64>(x: T) -> T {
         if x <= from_f64(0.04045) {
-            x / from_f64(12.92)
+            x * from_f64::<T>(12.92).recip()
         } else {
-            ((x + from_f64(0.055)) / from_f64(1.055)).powf(from_f64(2.4))
+            ((x + from_f64(0.055)) * from_f64::<T>(1.055).recip()).powf(from_f64(2.4))
```

Comment:
It seems like in some cases there's a variable being divided by a constant, this `recip` trick helps.